database/multistmt/parse.go

```diff
@@ -2,40 +2,26 @@
 package multistmt
 
 import (
-	"bytes"
 	"fmt"
 	"io"
 	"strings"
 
 	"github.com/pkg/errors"
 )
 
-// StartBufSize is the default starting size of the buffer used to scan and parse multi-statement migrations
-var StartBufSize = 4096
+// ParseBufSize is the buffer size for the multi-statement reader
+var ParseBufSize = 1024
```

Comment:
This seems awfully small, and we could potentially end up using more memory in the end, since we're not reusing the buffers, they will end up as garbage.